modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/store/get_store_value.es.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import fetch from './../../util/fetch.es';
+
+/**
+ * Gets the Store utility fetch value for 'get' case
+ * @param key {!String}
+ * @param callback
+ * @review
+ */
+
+export default function getStoreValue(key, callback) {
```

Comment:
On start of each function, we should check parameter types. If they are not expected, we should throw an error. Each such error should be checked in a test.